async/kafka/group/group.go

```diff
@@ -122,6 +122,7 @@ func (c *consumer) Consume(ctx context.Context) (<-chan async.Message, <-chan er
 				closeConsumer(c.cg)
 				return
 			case consumerError := <-c.cg.Errors():
+				log.Warnf("closing consumer group %s", consumerError)
```

Comment:
If you follow this channel the error might be logged already. Crea to check?

Reply:
It is not, only part that is logged is the fact that it tries to consume in a consumer group that is already closed

Reply:
Could we find the reason for this error getting "lost" in order to fix the problem? Logging does not solve this actually.

Reply:
This error is propagated to processing of component which then propagates to run of component and there, there is no error handling aside from retrying. We can log it there if you want, however it will still be a log, and personally im in favor or handling it sooner rather than later

Reply:
I disagree. The error is lost which is the actual problem and not logging.